pykokkos/interface/views.py

```diff
@@ -547,14 +554,21 @@ def from_cupy(array) -> ViewType:
     return from_numpy(np_array, MemorySpace.CudaSpace, layout)
 
 
+def isHashable(obj):
+    try:
+        hash(obj)
+    except TypeError:
+        return False
+    return True
+
+
 # asarray is required for comformance with the array API:
 # https://data-apis.org/array-api/2021.12/API_specification/creation_functions.html#objects-in-api
-
 def asarray(obj, /, *, dtype=None, device=None, copy=None):
     # TODO: proper implementation/design
     # for now, let's cheat and use NumPy asarray() followed
     # by pykokkos from_numpy()
-    if obj in {pk.e, pk.pi, pk.inf, pk.nan}:
+    if isHashable(obj) and obj in {pk.e, pk.pi, pk.inf, pk.nan}:
```

Comment:
Why do we need to check if it's hashable?

Reply:
I assumed that an `asarray` function can get a `list` as input as well (we are sending a list to asarray fn at 1 place in naive bayes example). Since lists are not hashable I believe, this code breaks with the following error

``` shell
File "/pykokkos1/pykokkos/interface/views.py", line 570, in asarray
    if obj in {pk.e, pk.pi, pk.inf, pk.nan}:
TypeError: unhashable type: 'list'
```